mega-zep-frontend/src/main/angular/frontend/src/app/modules/monthly-report/display-monthly-report/display-monthly-report.component.scss

```diff
@@ -29,8 +26,13 @@
   max-width: 25%;
 }
 
-// FIXME GAJ: Bug where checkbox changes color to black after visiting Mitarbeiteruebersicht(possible fix - create custom theme)
-::ng-deep .mat-checkbox-checked.mat-accent .mat-checkbox-background, .mat-checkbox-indeterminate.mat-accent .mat-checkbox-background, .mat-accent .mat-pseudo-checkbox-checked, .mat-accent .mat-pseudo-checkbox-indeterminate, .mat-pseudo-checkbox-checked, .mat-pseudo-checkbox-indeterminate {
+::ng-deep
```

Comment:
we should find a better way for this, ::ng-deep is deprecated ... see https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

Reply:
After some research I was not able to find an alternative to ng-deep. I know that it is not recommended but since Angular works with a shadow DOM, I am not sure what I can use instead.